CHANGELOG.tpl

```diff
@@ -7,11 +7,23 @@ Platform support legends:
     * Note: the papercuts will majorly be with the build process. Things might be very rough to integrate as no polish at all has been given yet.
 * ❌ = tier 3 support. It doesn't work just yet, but we plan to make it work.
 
+## [1.0.2] - 2024-08-24
+
+<details>
+    <summary>git-conventional changelog</summary>
+{{git-cliff tag="v1.0.2" unreleased=true}}
+</details>
+
+* added new tests for cross signed certificate chains
+* releases will use locked dependencies' versions
+* fixed a bug where retrieving the clients for multiple users would not return the full list
+* junit.xml should be added again to the release
```

Comment:
I noticed that in past releases we used proper sentences for each item, e.g.
```
* **[BREAKING]** The TypeScript wrapper now represents enums as their `number` discriminant instead of their `string` representation
```

Some of them with a dot at the end, some without. It would be nice to agree on a convention.